src/lib/components/SeminarAnmeldung/SeminarAnmeldung.svelte

```diff
@@ -1,5 +1,7 @@
 <script lang="ts">
   import SeminarCard from '$lib/components/SeminarCard/SeminarCard.svelte';
+  import DefaultCard from '$lib/components/DefaultCard/DefaultCard.svelte';
+
```

Comment:
```suggestion
```

Where did `DefaultCard` come from? I don't see it anywhere being used. Is it a left-over from some try/error?